frontendUiRootImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMachetePanel.java

```diff
@@ -60,11 +58,11 @@ public GitMachetePanel(Project project) {
   public @Nullable Object getData(String dataId) {
     return Match(dataId).of(
         typeSafeCase(DataKeys.KEY_BRANCH_LAYOUT_WRITER, branchLayoutWriter),
+        typeSafeCase(DataKeys.KEY_GRAPH_TABLE, graphTable),
```

Comment:
Maybe actually graph table (same for vcsRootComboBox) can be retrieved just directly from the project service instead of indirectly via `DataProvider`s like `GitMachetePanel`?

Reply:
resolved by https://github.com/VirtusLab/git-machete-intellij-plugin/pull/342